x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/import_summary/failures.tsx

```diff
@@ -56,11 +57,11 @@ export class Failures extends Component<Props, State> {
         }
         paddingSize="m"
       >
-        <div className="failure-list">
+        <div css={{ maxHeight: '200px', overflowY: 'auto' }}>
```

Comment:
Same question as above. Reformat for Emotion?

```suggestion
        <div css={css`
          ${logicalCSS('max-height', '200px')}
          overflow-y: auto;
        `}>
```